src/open_inwoner/search/resources/__init__.py

```diff
@@ -1 +1 @@
-from .resources import SynonymResource
+from .resources import SynonymResource  # noqa
```

Comment:
You can avoid the error from flake8 without `noqua` by explicitly declaring the class as part of the module interface:

`__all__ = [SynonymResource]`

Similarly for the othe imports in `__init__` files.

See the relevant section in [PEP 8](https://peps.python.org/pep-0008/#public-and-internal-interfaces)

Reply:
I'll change this :+1: